### ISSUE TITLE
Cannot find the application after installation

### ISSUE THREAD

**Issue by ssine** [EKA2L1/Dumber#2]
I hacked my N97 and installed the Dumber.sis successfully.
Then I can find Dumber.exe in C:\sys\bin\ but I cannot open it.
Also, I cannot find the entry of Dumber in the Menu.

**Comment by pent0** [comment]
> Then I can find Dumber.exe in C:\sys\bin\ but I cannot open it.

That's weird. I think if there's no memory card the app will crash. The Z drive package is hardcoded to be dumped in E drive. Can you check that?

> Also, I cannot find the entry of Dumber in the Menu.

It's supposed to be in a folder called "RnD Tools" on main menu. Have you checked that?

**Comment by ssine** [comment]
I do have an E drive, which is the phone's internal memory and has enough spare space.
I have checked all folders under main menu, but found nothing called "RnD Tools".

I just found that the sis files provided in repo and release page are earlier than the menu entry commit, are they updated? I have no idea on how to build the app.
BTW, are ".exe" files supported executables in symbian? Do they have any relation with windows exe files? I think the file explorer is complaining that exe is not a supported file and thus cannot be opened.
